docs/getting-started/getting-started-with-notebooks.rst

```diff
@@ -0,0 +1,164 @@
+Notebooks
+=========
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. note:: Dockstore Notebooks is currently in beta mode. Also note that the screenshots below were taken on our staging site.
+
+Tutorial Goals
+--------------
+
+- Get familiarized with the concept of notebooks on Dockstore
+- Learn the basics of creating a notebook using a .dockstore.yml file
+- Learn how to register a notebook
+- Publish your notebook to Dockstore
+
+Notebooks are documents that can include code, text, images, and other media resources. They can be executed in notebook environments like Google Colab and JupyterLab.
+This document will outline what is needed to register, update, and publish a notebook onto Dockstore. Please keep in mind that notebook functionality is currently in beta mode.
+
+Overview
+--------
+
+In this tutorial, you will register a notebook using the Dockstore GitHub App, which automatically syncs your notebook from GitHub to Dockstore. More information about the Dockstore GitHub App can be found :doc:`here </getting-started/github-apps/github-apps-landing-page>`.
+
+Dockstore Notebooks takes advantage of Google Colab's integration with GitHub. The integration allows you to develop your notebook in Google Colab and push your changes to GitHub from Google Colab. With the Dockstore GitHub App installed on your repository, any edits made to your notebook from Google Colab will automatically be synced to Dockstore.
+
+Setting up the Google Colab and GitHub integration
+--------------------------------------------------
+
+To get started, you will need a Google Colab account and a GitHub account.
+
+In your Google Colab account, link your Google Colab account to your GitHub account. Click on the Settings gear icon in the upper right corner, then select GitHub in the dialog box. Click on Authorise with GitHub and authorise Google Colab to access your GitHub account.
```

Comment:
`Authorise` -- at one point I believe we decided to standardize on American spelling instead of Canadian/UK, for better or worse :). This one would be tricky though since you have an accompanying screenshot with Canadian spelling, so I don't know.